Library/Homebrew/cask/audit.rb

```diff
@@ -785,8 +785,14 @@ def audit_https_availability
 
       return unless cask.homepage
 
+      user_agents = if cask.tap&.audit_exception(:simple_user_agent_for_homepage, cask.token)
```

Comment:
I don't think `audit_exception` is quite the right fit for this. Better have some attribute on e.g. the `cask.homepage` or `cask.url` similar to indicate that this is needed?

Reply:
I totally agree with you, but that would be a significant change.
This audit exception is analogous to the existing `secure_connection_audit_skiplist` which is a workaround for another kind of networking issues.
Anyway, the CDN failure is already affecting Microsoft Office users, so quick fix would be better than a profound fix at a later date.

Reply:
Re-reading this again: given it's audit-specific and, as you said, we have `secure_connection_audit_skiplist`: yes, I agree with you, let's just use this as-is.

Thanks @unshare for the polite and reasoned argument!